app/src/main/java/org/simple/clinic/home/overdue/OverdueAppointment.kt

```diff
@@ -18,7 +18,7 @@ import java.util.UUID
 
 @DatabaseView(
     """
-      SELECT P.fullName, P.gender, P.dateOfBirth, P.age_value, P.age_updatedAt,
+      SELECT P.fullName, P.gender, P.dateOfBirth, P.age_value, P.age_updatedAt, P.assignedFacilityId patientAssignedFacilityUuid,
```

Comment:
Since the view has changed, we'll need a database migration to drop the current view and replace it with the updated view.

Reply:
Ah right, forgot about that.

Reply:
Done